packages/gdl-frontend/components/modules/PaginationDot.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const styles = {
+  root: {
+    height: 18,
+    width: 18,
+    cursor: 'pointer',
+    border: 0,
+    background: 'none',
+    padding: 0
+  },
+  dot: {
+    backgroundColor: '#e4e6e7',
+    height: 9,
+    width: 9,
+    borderRadius: 6,
+    margin: 3
+  },
+  active: {
+    backgroundColor: '#0277bd'
+  }
+};
+
+class PaginationDot extends React.Component {
+  handleClick = event => {
+    this.props.onClick(event, this.props.index);
+  };
+
+  render() {
+    const { active } = this.props;
+
+    let styleDot;
+
+    if (active) {
+      styleDot = Object.assign({}, styles.dot, styles.active);
+    } else {
+      styleDot = styles.dot;
+    }
+
+    return (
+      <button type="button" style={styles.root} onClick={this.handleClick}>
+        <div style={styleDot} />
+      </button>
+    );
+  }
+}
+
+PaginationDot.propTypes = {
```

Comment:
`PropTypes` er en annen form for å sette type. Vi bruker `Flow`. Spørr gjerne Linn om hvordan dette gjøres :)